src/github.rs

```diff
@@ -1076,6 +1076,80 @@ impl Issue {
             .await?;
         Ok(())
     }
+
+    /// Checks if the PR's parent commit is too old.
+    ///
+    /// This determines if a PR needs updating by examining the first parent of the PR's head commit,
+    /// which typically represents the base branch commit that the PR is based on.
+    ///
+    /// If this parent commit is older than the specified threshold, it suggests the PR
+    /// should be updated/rebased to a more recent version of the base branch.
+    ///
+    /// Returns:
+    /// - Ok(Some(days_old)) - If parent commit is older than the threshold
+    /// - Ok(None) - If not a PR, can't get commit details, or parent is within threshold
+    /// - Err(...) - If an error occurred during processing
+    pub async fn is_parent_commit_too_old(
```

Comment:
This logic is IMO too specialized to live in `Issue` directly. I would just put it in `behind_master.rs`.